.github/workflows/test-and-release.yml

```diff
@@ -65,9 +55,9 @@ jobs:
             @semantic-release/git \
             @semantic-release/changelog \
             --no-progress --no-package-lock --no-save
-
       - name: Release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npx semantic-release
+
```

Comment:
Extra newline?

Reply:
yeah, it's a `cp` from existing ones and the newline is there too, gonna roll with it for now